src/firefly/js/templates/lightcurve/LcPeriodogram.jsx

```diff
@@ -256,7 +257,7 @@ class PeriodogramOptionsBox extends Component {
                         </button>
                         <br/>
                         <div style={{marginTop: 10}}>
-                            {'Fields of "Period Min", "Period Max" and "Step Size" are allowed to be blank.'}
+                            {'Leave the fields blank to use default values.'}  <HelpIcon helpId={'periodogram.options'}/>
```

Comment:
I meant, it should not hard to convert it to FormPanel when we have another chance. :)